src/datasources/system/SystemDataSource.test.ts

```diff
@@ -114,6 +114,28 @@ test('queries for system variable values - single workspace', async () => {
   expect(backendSrv.fetch.mock.lastCall?.[0].data).toHaveProperty('filter', 'workspace = "1"');
 });
 
+test('attempts to replace variables in metadata query', async () => {
+  const workspaceVariable = '$workspace';
+  backendSrv.fetch.mockReturnValue(createFetchResponse({ data: fakeSystems.map(({ id, alias }) => ({ id, alias })) }));
```

Comment:
```suggestion
  backendSrv.fetch.mockReturnValue(createFetchResponse({ data: fakeSystems }));
```

It doesn't really matter in this test, but this is the more correct response for the metadata query.